src/docs/ui-deployment.md

```diff
@@ -24,12 +24,31 @@ To get started, you'll need to be running from a bash/zsh environment. If you ar
 
 This process will build the user interface container image on your workstation using Docker, upload the container image to your Azure subscription, and install an instance of the container in Azure Container Instances (ACI). You'll need to have Docker installed locally, as well as the Azure Bash CLI.
 
-From the "src" directory
-
+Log in using the Azure CLI
+```BASH
```

Comment:
Just a few more linting issues in this file.

Reply:
Thank you. I was abel to fix the issues in the areas I added.